src/drivers/uavcan_v1/templates/Header.j2

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
You can use the templates in Nunavut when I finish https://github.com/UAVCAN/nunavut/pull/113 (we're pretty close right now). Can we move your serialization logic to Nunavut as well?